path.go

```diff
@@ -8,6 +8,10 @@ import (
 )
 
 func zoneFromPath(host string, path string) (string, int, error) {
+	bl := "/internal"
+	if strings.Contains(path, bl) {
+		return "", 0, fmt.Errorf("path containing 'internal' is disallowed")
+	}
```

Comment:
Not sure if we want to check this here, as it would prevent a standard web redirect containing `/internal` in its path from being executed correctly. The place where you put it in `gometa` is enough, and catches it at the right moment, I would say.

Reply:
Thanks - I removed that unnecessary check